packages/ui/components/core/Footer.tsx

```diff
@@ -0,0 +1,91 @@
+import { Container, Title, Grid, Text, Stack, Group, createStyles } from '@mantine/core'
+
+import { Button } from './Button'
+import { SocialMediaIconButton } from './SocialMediaIconButton'
+import { BodyGrid } from '../layout'
+
+const useStyles = createStyles((theme) => ({
+	footer: {
+		backgroundColor: theme.other.colors.primary.footer,
+		['& a']: {
+			color: theme.colors.dark,
+		},
+	},
+}))
+
+const supportLinks = [
+	['Suggest an organization', '#'],
+	['Share feedback', '#'],
+	['Vetting process', '#'],
+	['Privacy statement', '#'],
+	['Anti-hate commitment', '#'],
+	['Digital accessibility', '#'],
+	['Disclaimer', '#'],
+]
+
+const connectLinks = [
+	['Inreach.org', '#'],
+	['Download the app', '#'],
+	['Subscribe to our newsletter', '#'],
+]
+
+const makeLinks = (links: string[][]) =>
+	links.map(([text, href]) => (
+		<Text key={text} component='a' href={href} fw={500}>
```

Comment:
Internal links should use the [Link](https://github.com/weareinreach/InReach/blob/IN-715-footer-component-redesigned/packages/ui/components/core/Link.tsx#L1) component rather than the basic `a` anchor tag 

```suggestion
		<Text key={text} component={Link} href={href} fw={500}>
```